### PR TITLE
change default port number to 7355

### PR DESCRIPTION
**Pull request by pjfanning**
relates to #208 

* old akka port default 2552 becomes 7355 (PEKK on a telephone keypad)
* old akka artery default of 25520 becomes 17355 (73550 > 65535 - max port value)

### CODE REVIEW

File: cluster-typed/src/test/scala/docs/org/apache/pekko/cluster/typed/BasicClusterExampleSpec.scala [hunk 1]
```diff
@@ -53,7 +53,7 @@ pekko {
   cluster {
     seed-nodes = [
       "pekko://ClusterSystem@127.0.0.1:2551",
```
Comment (nvollmar) [inline cluster-typed/src/test/scala/docs/org/apache/pekko/cluster/typed/BasicClusterExampleSpec.scala:55]: change this to 7354?

File: cluster/src/test/scala/org/apache/pekko/cluster/ClusterDomainEventPublisherSpec.scala [hunk 2]
```diff
@@ -52,18 +52,18 @@ class ClusterDomainEventPublisherSpec
...
   val aExiting = aLeaving.copy(status = Exiting)
   val aRemoved = aExiting.copy(status = Removed)
-  val bExiting = TestMember(Address(protocol, "sys", "b", 2552), Exiting)
+  val bExiting = TestMember(Address(protocol, "sys", "b", 7355), Exiting)
   val bRemoved = bExiting.copy(status = Removed)
-  val cJoining = TestMember(Address(protocol, "sys", "c", 2552), Joining, Set("GRP"))
+  val cJoining = TestMember(Address(protocol, "sys", "c", 7355), Joining, Set("GRP"))
   val cUp = cJoining.copy(status = Up)
   val cRemoved = cUp.copy(status = Removed)
   val a51Up = TestMember(Address(protocol, "sys", "a", 2551), Up)
```
Comment (nvollmar) [inline cluster/src/test/scala/org/apache/pekko/cluster/ClusterDomainEventPublisherSpec.scala:64]: 7354?

**Review by nvollmar — APPROVED** [review]
lgtm

File: cluster/src/test/scala/org/apache/pekko/cluster/ClusterDomainEventPublisherSpec.scala (hunk 2 above)
Comment (pjfanning) [reply cluster/src/test/scala/org/apache/pekko/cluster/ClusterDomainEventPublisherSpec.scala]: done

**Comment by He-Pin** [comment]
Update the doc about why we choose this port too？

**Comment by pjfanning** [comment]
> Update the doc about why we choose this port too？

I've added a new commit to the PR branch with an update to the migration guide

**Pull request merged**